[2022-06-13 09:51:07,221] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: a_call_dag.end manual__2022-06-13T09:48:43.605166+00:00 [queued]>
[2022-06-13 09:51:07,308] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: a_call_dag.end manual__2022-06-13T09:48:43.605166+00:00 [queued]>
[2022-06-13 09:51:07,309] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 09:51:07,310] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-13 09:51:07,320] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 09:51:07,440] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): end> on 2022-06-13 09:48:43.605166+00:00
[2022-06-13 09:51:07,484] {standard_task_runner.py:52} INFO - Started process 17409 to run task
[2022-06-13 09:51:07,507] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'a_call_dag', 'end', 'manual__2022-06-13T09:48:43.605166+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/call_dag.py', '--cfg-path', '/tmp/tmp714sbfun', '--error-file', '/tmp/tmpeld64en1']
[2022-06-13 09:51:07,520] {standard_task_runner.py:80} INFO - Job 128: Subtask end
[2022-06-13 09:51:08,092] {task_command.py:370} INFO - Running <TaskInstance: a_call_dag.end manual__2022-06-13T09:48:43.605166+00:00 [running]> on host 8ef6f3d8e77e
[2022-06-13 09:51:08,682] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=a_call_dag
AIRFLOW_CTX_TASK_ID=end
AIRFLOW_CTX_EXECUTION_DATE=2022-06-13T09:48:43.605166+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-13T09:48:43.605166+00:00
[2022-06-13 09:51:08,688] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-13 09:51:08,694] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo end']
[2022-06-13 09:51:08,745] {subprocess.py:85} INFO - Output:
[2022-06-13 09:51:08,748] {subprocess.py:92} INFO - end
[2022-06-13 09:51:08,752] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-13 09:51:08,843] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=a_call_dag, task_id=end, execution_date=20220613T094843, start_date=20220613T095107, end_date=20220613T095108
[2022-06-13 09:51:08,954] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-13 09:51:09,184] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
